### PR TITLE
[Core/OpenTelemetry] Fixed spelling errors

### PR DESCRIPTION
**Pull request by wonhyeongseo**
# Description
Closes https://github.com/Azure/azure-sdk-for-python/issues/22708

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Comment by xiangyan99** [comment]
#22708

File: sdk/core/azure-core-tracing-opentelemetry/tests/test_tracing_implementations.py [hunk 1]
```diff
@@ -188,4 +188,7 @@ def test_span_kind(self, tracer):
             assert wrapped_class.kind == SpanKind.INTERNAL
 
             with pytest.raises(ValueError):
+                # TODO: what should be done here?
```
Comment (kristapratico) [inline sdk/core/azure-core-tracing-opentelemetry/tests/test_tracing_implementations.py:191]: I think we can also move this one under tests path in the cspell.json.

**Review by kristapratico — COMMENTED** [review]
Yay thanks for your contribution! 😸

**Review by kristapratico — APPROVED** [review]

**Comment by kristapratico** [comment]
The cosmos failures are unrelated to this PR so I'm going to merge this. Thanks for your contribution! 🐱

**Pull request merged**